src/tests/nativeaot/SmokeTests/ComWrappers/ComWrappers.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+    <!-- Look for MULTIMODULE_BUILD #define for the more specific incompatible parts -->
+    <NativeAotMultimoduleIncompatible Condition="'$(Optimize)' == 'true'">true</NativeAotMultimoduleIncompatible>
```

Comment:
I think this will just work. The debug leg is already succeeding in multimodule mode and I don't think this has any optimized code problems.

```suggestion
```